### DISCUSSION TITLE
Second release

### DISCUSSION THREAD

**Discussion by diazona** [diazona/setuptools-pyproject-migration#120] [General]
I'm thinking we should target a second release in a couple weeks or so - what say you @sjlongland ? We'd just have to make a bit of a push to get a couple of issues resolved and then everything attached to that 0.2 milestone would be done.

**Comment by sjlongland** [comment]
That sounds reasonable.  I'm back at work now, but quite a few important fixes and improvements are already merged, so 0.2 should not be the long slog that 0.1 was. :-)

**Comment by diazona** [comment]
Whoops, so actually I had some other things to deal with and haven't had the energy to finish working on #105. It turned out to be quite a bit more complex than I expected. It's getting close, though, and I have a holiday weekend coming up so I should be able to wrap that up soon.

**Comment by diazona** [comment]
I set the due date to December 15, now that I completed my work on #105. (We can release before that, too, if everything is ready)

**Comment by diazona** [comment]
@sjlongland FYI I had the misfortune of experiencing a drive failure on my personal desktop earlier this week, and I lost a lot of data. I think I should be able to recover in time to meet our planned release schedule, but it'll take a couple more days before I can do much of any development.

**Reply by diazona** [reply]
Fortunately I managed to recover almost everything :smile: My computer is still a bit messed up, but fortunately no more significant delays.

**Reply by sjlongland** [reply]
No problems… yeah, tried replying to this via email (which used to work) but I'm yet to see my earlier reply show up here.

Sucks when a drive fails, thankfully it's not as frequent occurrence as it once was, but it still happens.  Got two 2TB laptop drives that are completely dead I use as paperweights (one is reporting failures in SMART, the other won't even detect).

Good to hear you managed to recover most of your data.  I hate data loss.

No rush on the release.  Take whatever time you need to get your workstation sorted out. :-)

**Reply by diazona** [reply]
Haha yup it's one of those things that are probably like battle scars for tech people. I definitely got another drive to add to the paperweight collection (though it's a small SSD so not even that good as a paperweight!), but I lucked out in that even after the failure it was working well enough to decrypt and capture almost a complete drive image, and from that I got pretty much all the important stuff from my home directory back. So I will have more work to do cleaning this up over the coming weeks, but for now I'm all set.

**Reply by sjlongland** [reply]
You haven't owned a "real" computer if you haven't experienced data loss. :-)

**Comment by diazona** [comment]
Hmm, so apparently we have [a problem to fix](https://github.com/diazona/setuptools-pyproject-migration/actions/runs/7242100985/job/19727074342) before actually releasing v0.2. I'll make an issue out of it shortly.

(also this definitely makes me want to prioritize changing the release workflow to ensure that the code builds and passes tests before tagging and publishing the release, but that will be something to leave for after we get this one finalized)

**Comment by diazona** [comment]
And it's done! https://pypi.org/project/setuptools-pyproject-migration/
